database/insertData/03_template_E_drugRegistration.js

```diff
@@ -0,0 +1,610 @@
+                      title: "Optinal information"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      parameters: {
+                        text: "Optinal information"
+                      }
+                    }
+                    {
+                      code: "S1Q21"
+                      index: 300
+                      title: "Shelf life (months)"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      parameters: {
+                        label: "Shelf life (months)"
+                      }
+                      isRequired: false
+                    }
+                    {
+                      code: "S1Q22"
+                      index: 310
+                      title: "Comments"
+                      elementTypePluginCode: "shortText"
+                      category: QUESTION
+                      parameters: {
+                        label: "Comments"
+                      }
+                      isRequired: false
+                    }
+                  ]
+                }
+              }
+              {
+                id: 1008
+                code: "S2"
+                title: "Ingredients"
+                index: 1
+                templateElementsUsingId: {
+                  create: [
+                    {
+                      code: "S2Info1"
+                      index: 0
+                      title: "Product Intro"
+                      elementTypePluginCode: "textInfo"
+                      category: INFORMATION
+                      parameters: {
+                        text: "In this section, we require the INGREDIENTS for **Drug registered**"
+                      }
+                    }
+                    {
+                      code: "S2Q1"
+                      index: 1
+                      title: "File upload: Ingredients"
+                      elementTypePluginCode: "fileUpload"
+                      category: QUESTION
+                      isRequired: false
+                      parameters: {
+                        label: "Upload ingredients list"
+                        description: "Only 1 file allowed, no other restrictions"
+                        fileExtensions: ["pdf", "doc", "docx", "xls"]
+                        fileCountLimit: 1
+                      }
+                    }
+                  ]
+                }
+              }
+              {
+                id: 1009
+                code: "S3"
+                title: "Product images"
+                index: 2
+                templateElementsUsingId: {
+                  create: [
+                    {
+                      code: "S3Q1"
+                      index: 0
+                      title: "File upload: Samples"
+                      elementTypePluginCode: "fileUpload"
+                      category: QUESTION
+                      isRequired: false
```

Comment:
Yes.

One thing I noticed is that even for `isRequired`  file uploads if you try to load one not allowed (e.g. wrong format) it counts as valid - need to create an issue for that. I'll add a task to the board, but just so you know there is a fix required on that.

Reply:
Ah yeah, that's a good point. Good spotting. I made an issue for another little tweak I noticed, so maybe they can be included together since they're both pretty tiny fixes: https://github.com/orgs/openmsupply/projects/6#card-57562132